lib/Reader/reader.cpp

```diff
@@ -5211,6 +5211,8 @@ ReaderBase::rdrCall(ReaderCallTargetData *Data, ReaderBaseNS::CallOpcode Opcode,
 
   std::vector<CallArgType> &ArgumentTypes =
       Data->CallTargetSignature.ArgumentTypes;
+
+  // Vector is being initialized for the first time as it was skipped in init.
```

Comment:
I don't understand this comment. How is this case different from the one where you changed code to use .resize?